src/get-started/java-binding.md

```diff
@@ -160,68 +163,56 @@ describes their use.
     `SparseListIndex` is not yet supported in Java. Let us know if it may be
     useful for you!
 
-Collections work with a database view – either `Snapshot`, which is read-only
-and represents the database state as of the latest committed block;
+Collections are instantiated using a database view. The database view
+works as an index factory. The views may be based on a `Snapshot` — a read-only
+view corresponding to the database state as of the latest committed block;
 or `Fork`, which is mutable and allows performing modifying operations. The
 database view is provided by the framework: `Snapshot` can be
 requested at any time, while `Fork` – only when the transaction is executed. The
 lifetime of these objects is limited by the scope of the method to which they
 are passed to.
+<!-- todo: Do we need to describe the Prefixed etc? -->
```

Comment:
IMO, describing `Prefixed` etc. (except for maybe a brief mention) would be redundant. These types are already described in the *MerkleDB* article. IMO, we should avoid redundant content. (Can't help but notice that some sections, such as "Transaction Lifecycle" have little to do with Java and would look better in the "general" articles.)